### PR TITLE
fix: attendeeframent progress bar

### LINKED ISSUE #956: Progress bar shown when internet connection off in attendee fragment
**Describe the bug**

The progress bar is shown even before clicking on the register button when the internet connection is off.

**To Reproduce**

1. Keep the internet connection off.
2. Go to the attendee fragment.

**Expected behavior**

A Snackbar or toast to be displayed showing "Please check your internet connection".

**Logs**

**Screenshots**

**Smartphone Info:**

|               | |
|---------------|-|
|Device         |Lenovo P2|
|Android Version| Nougat 7.0  |

**Additional context**

@nikit19 @liveHarshit @iamareebjamal  your views?

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by angmas1**
Fixes #956 

Changes: Fixed bug where progressBar would show indefinitely if internet connection is lost in attendee fragment. Added snackbar when internet connection lost and when register button is pressed in attendeefragment. Moved isNetworkConnected to Utils. 

Screenshots for the change:
ezgif com-video-to-gif

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt [hunk 1]
```diff
@@ -362,39 +363,40 @@ class AttendeeFragment : Fragment() {
...
-                    event = eventId)
-                attendees.add(attendee)
-            } else {
-                attendees.addAll(attendeeRecyclerAdapter.attendeeList)
-            }
-
-            if (attendeeViewModel.areAttendeeEmailsValid(attendees)) {
-                val country = if (country.text.isEmpty()) country.text.toString() else null
-                attendeeViewModel.createAttendees(attendees, country, paymentOptions[selectedPaymentOption])
+            if (isNetworkConnected(context)) {
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt]: If it is not connected, just return

Comment (angmas1) [reply app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt]: okay

Comment (angmas1) [reply app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt]: done

**Comment by angmas1** [comment]
@nikit19 resolved

**Pull request merged**